# Question 1
Two gamblers decided to play a game involving rolling a *fair* die. Player $A$ agreed to pay Player $B$ $\$10$ as an “entry fee” to play. The game consists of Player $A$ rolling a die until he rolls an $1$ for the first time. An example of a possible sequence of rolls in this game is ${(2, 6, 3, 5, 1)}$, with the number of rolls $N = 5$. Assume the rolls are independent

### A) Player $A$ wins the game if it takes at least 3 rolls for the first 1 to show up. Find out the probability that player A wins one round of the game.(3 marks)

This is a classic geometric distribution problem. Since this is a fair die, the probability of getting a success (getting a 1) would be 1 in 6 or $p=\frac{1}{6}=0.1\bar{6}$

$$
X \text{~} Geo(0.1666...)
$$

Now we have to find the probability of winning the first round by delaying the success until 3 or more. i.e. $P(X\geq 3)$ but it could take infinite roles to get a 1, so instead I'll find $1-P(X<3)$

$$
F(x)=P(X\leq x)=1−(1−p)^x
$$
In this case, $x = 2$

In [1]:
p = 1/6
x = 2

ans = 1-(1-p)^x
paste("The probability of player A winning 1 round is", round(1-ans,6)*100,'%')

[1] "The probability of player A winning 1 round is 69.4444 %"

### B) For each round of the game, if Player A wins according to the condition in part (a), then he gets $\$30$ from Player B. Find the expected value of Player A’s profit. Is this game a good deal for Player A? Briefly explain your answer. (3 marks)

On average, Player A has a 69\% chance of winning a game. If he wins (which 69\% of the time he does) he wins 20 dollars (30 - 10 for entry fee).  If he loses (which he has a 31\% chance of doing), he lost 10 dollars. Just from this number alone he seems to be winning. But how much on average?

In [2]:
win = 1-ans
lose = 1-win

winnings = win*20 + lose*-10
print(paste('$',winnings))

[1] "$ 10.8333333333333"


his expected winnings on average is $\$10.83$ so the game is a pretty sweet deal, although a bit shady. 

### C) Player A decides to play the game for 100 rounds. Find the expectation and variance of the number of games that player A wins according to the condition in part (a). (4 marks)

This seems like a binomial problem since we are given a number of trials. 

The mean number of times he will win is $E(X)=np$:

In [3]:
n=100
p = win
expectation = n*p
expectation

[1] 69.44444

And the variance is calculated via $np(1-p)$

In [4]:
variance = n*p*(1-p)
variance

[1] 21.21914